src/ImageSharp/Formats/Jpeg/Components/Decoder/ColorConverters/JpegColorConverter.FromYCbCrVector4.cs

```diff
@@ -87,7 +74,7 @@ internal static void ConvertCore(in ComponentValues values, Span<Vector4> result
                     tmp.MultiplyInplace(1.772F);
                     b.AddInplace(ref tmp);
 
-                    if (Vector<float>.Count == 4)
+                    if (SimdUtils.HasVector4)
```

Comment:
For perf, we should remove this branch now and use `RoundAndDownscalePreVector8` only. `IsAvailable` has already done the capability check by this time.

Reply:
Good catch, I removed the branch